### PR TITLE
[tests-only] Add test to share deleted resource

### PR DESCRIPTION
**Pull request by grgprarup**
## Description
Added Scenarios:
- Scenario: send share invitation to user for deleted file
- Scenario: send share invitation to group for deleted file

## Related Issue

- Part of https://github.com/owncloud/ocis/issues/7947

## How Has This Been Tested?

- Locally
- :robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiSharingNg/shareInvitations.feature [hunk 1]
```diff
@@ -1664,3 +1664,95 @@ Feature: Send a sharing invitations
       | Space Editor     |
       | Co Owner         |
       | Manager          |
+
+
+  Scenario: send share invitation to user for deleted file
```
Comment (nabim777) [inline tests/acceptance/features/apiSharingNg/shareInvitations.feature:1669]:
```suggestion
  Scenario: send a share invitation to the user for a deleted file
```

File: tests/acceptance/features/apiSharingNg/shareInvitations.feature [hunk 2]
```diff
@@ -1664,3 +1664,95 @@ Feature: Send a sharing invitations
...
+                "enum": ["stat: error: not found: "]
+              }
+            }
+          }
+        }
+      }
+      """
+
+
+  Scenario: send share invitation to group for deleted file
```
Comment (nabim777) [inline tests/acceptance/features/apiSharingNg/shareInvitations.feature:1711]:
```suggestion
  Scenario: send a share invitation to the group for a deleted file
```

File: tests/acceptance/features/apiSharingNg/shareInvitations.feature [hunk 3]
```diff
@@ -1664,3 +1664,95 @@ Feature: Send a sharing invitations
...
+    Given user "Carol" has been created with default attributes and without skeleton files
+    And group "grp1" has been created
+    And the following users have been added to the following groups
+      | username | groupname |
+      | Brian    | grp1      |
+      | Carol    | grp1      |
+    And user "Alice" has uploaded file with content "to share" to "textfile1.txt"
+    And we save it into "FILEID"
+    And user "Alice" has deleted file "textfile1.txt"
+    When user "Alice" sends the following share invitation with file-id "<<FILEID>>" using the Graph API:
```
Comment (nabim777) [inline tests/acceptance/features/apiSharingNg/shareInvitations.feature:1721]:
```suggestion
    When user "Alice" sends the following share invitation with the file-id "<<FILEID>>" using the Graph API:
```

File: tests/acceptance/features/apiSharingNg/shareInvitations.feature (hunk 1 above)
Comment (saw-jan) [reply tests/acceptance/features/apiSharingNg/shareInvitations.feature:1669]: let's keep it like above scenarios

File: tests/acceptance/features/apiSharingNg/shareInvitations.feature (hunk 3 above)
Comment (grgprarup) [reply tests/acceptance/features/apiSharingNg/shareInvitations.feature:1816]: An article is not needed here.

File: tests/acceptance/features/bootstrap/SharingNgContext.php [hunk 4]
```diff
@@ -146,8 +147,11 @@ public function sendShareInvitation(string $user, TableNode $table): ResponseInt
 		}
 		$spaceId = $space['id'];
 
+		// $fileId is used for sharing deleted files
```
Comment (saw-jan) [inline tests/acceptance/features/bootstrap/SharingNgContext.php:150]:
```suggestion
		// $fileId is used for trying to share deleted files
```

**Review by SagarGi — APPROVED** [review]
other LGTM :+1:

**Review by saw-jan — APPROVED** [review]

**Comment by SagarGi** [comment]
@grgprarup please remove tag `full-ci` and push once again. The CI is failing due to this issue for now. https://github.com/owncloud/ocis/issues/8332

**Pull request merged**